sgd/sgd-aa-oas3.yml

```diff
@@ -0,0 +1,577 @@
+      summary:
+        Returns the document with the terms and conditions related to this
+        system
+      responses:
+        "200":
+          description: Ok
+          content:
+            text/html:
+              schema:
+                type: string
+                description: HTML page with the Terms of Service
+        "404":
+          $ref: "#/components/responses/404"
+      tags:
+        - public
+      x-spid-operation: # See §2.8
+        consentRequired: "N"
+        offlineAccess: true
+        offlineAccessDurationMax: 8640000
+
+  /delegation/list:
+    get:
+      operationId: get-delegation
+      summary:
+        Returns the list of the delegations related to the services (or class of
+        services) provided by the authenticated RP and having the current user
+        as delegator
+      security:
+        - dpopToken: []
+          dpopProof: []
+      parameters:
+        - name: DPoP
+          in: header
+          required: true
+          description: DPoP proof
+          schema:
+            "$ref": "#/components/schemas/DPoP"
+      x-spid-operation: # See §2.8
+        consentRequired: "SP"
+        spidLevel: SPID2
+        offlineAccess: false
+        offlineAccessDurationMax: 0
+      tags:
+        - protected
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DelegationList"
+              example:
+                delegation_list:
+                  - id: 1A00001AA21
+                    https://attributes.eid.gov.it/fiscal_number: TINIT-RSSMR*********I
+                  - id: 1A00002AA21
+                    https://attributes.eid.gov.it/fiscal_number: TINIT-ABCXX*********Z
+                  - id: 1A00004AA21
+                    https://attributes.eid.gov.it/fiscal_number: TINIT-DEFYY*********W
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+          #The requested service (or class of services) does not exist
+        "429":
+          $ref: "#/components/responses/429"
+        "500":
+          $ref: "#/components/responses/500"
+
+  /delegation/list/{service}:
+    get:
+      description: fdfd
+      operationId: get-delgation-service
+      summary:
+        "Returns the list of the delegations having the current user as
+        delegator and related to the services (or class of services) provided by
+        the authenticated RP and identifed from the service component in the URL"
+      security:
+        - dpopToken: []
+          dpopProof: []
+      x-spid-operation: # See §2.8
+        consentRequired: "SP"
+        spidLevel: SPID2
+        offlineAccess: false
+        offlineAccessDurationMax: 0
+      tags:
+        - protected
+      parameters:
+        - in: path
+          name: service
+          required: true
+          schema:
+            type: string
```

Comment:
```suggestion
            type: string
            format: uuid
```

Reply:
Effettivamente volevo aggiungerlo, ma non sono riuscito a trovare un riferimento allo standard  OAS3 sul formato uuid. Mi puoi dare qualche reference?

Reply:
Trovato [qui](https://swagger.io/docs/specification/data-models/data-types/) si dice:

_"However, format is an open value, so you can use any formats, even not those defined by the OpenAPI Specification, such as:
email
uuid
uri
hostname
ipv4
ipv6
and others"_